lib/matplotlib/collections.py

```diff
@@ -2047,6 +2048,54 @@ def set_paths(self):
         self._paths = self._convert_mesh_to_paths(self._coordinates)
         self.stale = True
 
+    def set_array(self, A):
+        """
+        Set the data values.
+        
+        Parameters
+        ----------
+        A : (M, N) array-like or M*N array-like
+            If the values are provided as a 2D grid, the shape must match the
+            coordinates grid. If the values are 1D, they are reshaped to 2D.
+            
+            M, N follow from the coordinates grid, where the coordinates grid
+            shape is (M, N) for 'gouraud' *shading* and (M+1, N+1) for 'flat'
+            shading. 
```

Comment:
```suggestion
            shading.
```

Reply:
I thought I fixed this locally, my apologies